ansible/environments/mac/group_vars/all

```diff
@@ -1,7 +1,7 @@
 whisk_version_name: mac
-nginx_conf_dir: /Users/Shared/nginx
-consul_conf_dir: /Users/Shared/consul
-whisk_logs_dir: /Users/Shared/wsklogs
+nginx_conf_dir: /tmp/nginx
```

Comment:
I think this is because I copied environment file from `local` environment at first time.
I will update it.